docs/deploying_csi_vsphere_with_rbac.md

```diff
@@ -93,15 +121,15 @@ datacenters = "list of datacenters where Kubernetes node VMs are present"
         # datacenters will be used from Global section.
 ```
 
-Configure your vsphere.conf file and create a `configmap` of your settings using the following command:
+Configure your `vsphere.conf` file and create a `configmap` of your settings using the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create configmap cloud-config --from-file=vsphere.conf --namespace=kube-system
+$ kubectl create configmap cloud-config --from-file=vsphere.conf --namespace=kube-system
```

Comment:
`cloud-config` should be `csi-config`. I think this might even be a bug carried over from the old documentation. We should document the CSI driver using a different config (there are some use cases to support this).

Reply:
I left `cloud-config` in the YAML specifically because we showed creating the config map with this name. I can change it in both places, though.